test/integration/http/github-repo-enabled-payload-test.js

```diff
@@ -0,0 +1,85 @@
+const path = require('path')
+const test = require('ava')
+const uuid = require('uuid')
+const installationDao = require('~/src/dao/github/installation')
+const repositoryDao = require('~/src/dao/github/repository')
+const addToContext = require('~/test/util/addToContext')
+
+function randomInRange (min, max) {
```

Comment:
This is going to go away. I plan on adding lodash's `_.random` function to util. https://lodash.com/docs/4.17.4#random